doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -0,0 +1,171 @@
+Connect to bookmarked node            Right-click plug > `Connect Bookmark` > select node
+Insert node at connection             Drag node onto connection
+Auto-arrange selected nodes           :kbd:`Ctrl`-:kbd:`L`
+===================================== =============================================
+```
+
+
+### Navigation ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Shortcut or Gesture
+===================================== =============================================
+Pan                                   :kbd:`Alt`-click and drag
+Zoom                                  :kbd:`Alt`-right-click and drag
+"                                     Mouse wheel up/down
+Frame selected nodes                  :kbd:`F`
+Enter `Box` node (subgraph)	          :kbd:`↓`
+Leave `Box` node (subgraph)           :kbd:`↑`
+Search for nodes                      :kbd:`Ctrl`-:kbd:`F`
+Bookmark node                         Right-click on node
+===================================== =============================================
+```
+
+
+## _Viewer_ Tab ##
+
+> Note :
+> For the following controls and shortcuts, the cursor must hover inside the _Viewer_ tab.
+
+
+### 2D Images ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Shortcut/gesture
+===================================== =============================================
+Pan                                   :kbd:`Alt`-click and drag
+Zoom                                  :kbd:`Alt`-right-click and drag
+"                                     Mouse wheel up/down
+Frame image                           :kbd:`F`
+Center image at 1:1 scale             :kbd:`Home`
+Toggle red channel                    :kbd:`R`
+Toggle green channel                  :kbd:`G`
+Toggle blue channel                   :kbd:`B`
+Toggle alpha channel                  :kbd:`A`
+===================================== =============================================
+```
+
+### 3D Scenes ###
+
+```eval_rst
+========================================== =====================================
+Action			                           Shortcut/gesture
+========================================== =====================================
+Tumble                                     :kbd:`Alt`-click and drag
+Track                                      :kbd:`Alt`-middle-click and drag
+Dolly                                      :kbd:`Alt`-right-click and drag
+"                                          Mouse wheel up/down
+Expand selection                           :kbd:`↓`
+Fully expand selection                     :kbd:`Shift`-:kbd:`↓`
+Collapse selection                         :kbd:`↑`
+Frame selection                            :kbd:`F`
+Fit clipping planes to selection           :kbd:`Ctrl`-:kbd:`K`
+Frame and fit clipping planes to selection :kbd:`Ctrl`-:kbd:`F`
```

Comment:
Not in the original copy, but there are also 3d viewer controls via RMB on the background (todo with these clipping planes)